test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,27 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(100);
+const N = 5;
 var nbIntervalFired = 0;
-var timer = setInterval(function() {
+
+const keepOpen = setTimeout(() => {
+  console.error('[FAIL] Interval fired %d/%d times.', nbIntervalFired, N);
+  throw new Error('Test timed out. keepOpen was not canceled.');
+}, TEST_DURATION);
+
+const timer = setInterval(() => {
   ++nbIntervalFired;
-  if (nbIntervalFired === N)
+  if (nbIntervalFired === N) {
     clearInterval(timer);
+    timer._onTimeout = () => {
```

Comment:
@Fishrock123 Is this the change you had in mind or should I also not clear the keepOpen timeout to give the error a chance to happen.